appliance/postgresql/runner.go

```diff
@@ -83,14 +91,14 @@ func startLeader() (*exec.Cmd, <-chan struct{}) {
 }
 
 func register(attrs map[string]string) {
-	err := discoverd.RegisterWithAttributes(*serviceName, addr, attrs)
+	err := heartbeater.SetMeta(attrs)
 	if err != nil {
 		log.Fatalln("discoverd registration error:", err)
```

Comment:
The error and the function name could be changed to be a bit more accurate (since by this point, we've already registered on discoverd)